### ISSUE TITLE
YouTubeSearchResults.thumbnails typed as string in index.d.ts

### ISSUE THREAD

**Issue by mat3049** [MaxGfeller/youtube-search#22]
When attempting to access properties on thumbnails, an error occurs:

```
constructor(data: YouTubeSearchResults) {
    super()

    // This line gives a TS error "Property 'default' does not exist on type 'string'"
    let thumbnail = data.thumbnails.default.url
  }
```

However, this code will work as per [Thumbnails API docs](https://developers.google.com/youtube/v3/docs/thumbnails), since `thumbnails` is a nested object.

Working on a PR to resolve this.

**Comment by frankhale** [comment]
> Working on a PR to resolve this.

Thank you!!!

**Comment by MaxGfeller** [comment]
Fixed by #23
